batchflow/notifier.py

```diff
@@ -337,7 +337,7 @@ def create_description(self, iteration):
             name = container['name']
             if isinstance(source, (str, NamedExpression)):
                 value = container['data'][iteration]
```

Comment:
`value = getattr(value, 'tolist', lambda: value)()`
this will convert value to pure python scalar